src/components/config/config.py

```diff
@@ -18,16 +18,16 @@
 BASE_CURATION_FOLDER = DATA_FOLDER / "curation"
 
 
-if not os.path.exists(BASE_EXTRACTION_FOLDER):
-    os.mkdir(BASE_EXTRACTION_FOLDER)
-if not os.path.exists(BASE_CURATION_FOLDER):
-    os.mkdir(BASE_CURATION_FOLDER)
+# if not os.path.exists(BASE_EXTRACTION_FOLDER):
+#     os.mkdir(BASE_EXTRACTION_FOLDER)
+# if not os.path.exists(BASE_CURATION_FOLDER):
+#     os.mkdir(BASE_CURATION_FOLDER)
 
-ckpt = "icdar_19b2_v2.pth" if "cpu" in torch.__version__ else "icdar_19b2.pth"
+ckpt = "icdar_19b2_v2.pth" #if "cpu" in torch.__version__ else "icdar_19b2.pth"
```

Comment:
The config file is being used in the text extraction notebook, if we change its path then all the paths within the config file will have to be updated as well.